website/assets/js/pages/query-library.page.js

```diff
@@ -12,57 +13,72 @@ parasails.registerPage('query-library', {
 
   computed: {
     filteredQueries: function () {
-      return _.filter(this.queries, (query) => this.isIncluded(query.platforms, this.selectedPlatform) && this.isIncluded(query.purpose, this.selectedPurpose));
+      return _.filter(
```

Comment:
also no need to change this, but fyi this now exists as a native function in js: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter